spec/controllers/users_controller_spec.rb

```diff
@@ -11,7 +11,11 @@
   describe "POST create" do
     context "sending emails" do
       after {ActionMailer::Base.deliveries.clear}
-      before {ActionMailer::Base.deliveries.clear}
+      let(:charge) {double(:charge, successful?: true)}
+      before do
+        ActionMailer::Base.deliveries.clear
+        StripeWrapper::Charge.stub(:create).and_return(charge)
```

Comment:
This is the older `should` syntax which is no longer recommended by rspec's maintainers. The `expect` syntax for this is:

```ruby
expect(StripeWrapper::Charge).to receive(:create).and_return(charge)
```